usr/sbin/p11sak/p11sak.c

```diff
@@ -2781,7 +2852,8 @@ static CK_RV parse_file() {
     // open and parse file
     if (file_loc != NULL) {
         if ((fp = fopen(file_loc, "r")) == NULL) {
-            fprintf(stderr, "Failed to load %s: %s\n", file_loc, strerror(errno));
+            fprintf(stderr, "Cannot load config file from env variable %s: %s\n", file_loc, strerror(errno));
```

Comment:
Very misleading error message: "Cannot load config file from env variable /does/not/exist: No such file or directory".  Did you mean `P11SAK_DEFAULT_CONF_FILE` instead of the first `%s` (or even better before the `%s` and add the value in brackets).

Reply:
Argh ... good point: yes I should print the name of the env variable instead of the first %s, and as you suggest, also print the actual value

Reply:
Done.